thunder/dynamo/benchmark_utils.py

```diff
@@ -329,7 +345,27 @@ def get_pretty_time_str(a):
     return f"{a / time_scale:.3f} {time_unit}"
 
 
-def check_threshold_log(a: float, b: float, a_name: str, b_name: str, test_name: str, timer_name: str, rtol, atol):
+def get_pretty_memory_str(value):
+    """
+    Converts memory size in bytes to human readable string with appropriate unit suffix.
+
+    Args:
+        value (float): Memory size in bytes
+
+    Returns:
+        str: Memory size with appropriate unit (B, KB, MB, GB, TB, PB)
+    """
+    suffixes = ["B", "KB", "MB", "GB", "TB", "PB"]
+    suffix_idx = 0
+
+    while suffix_idx + 1 < len(suffixes) and value >= 1024.0:
+        value /= 1024.0
```

Comment:
So here's the thing: is a kilobyte 1000 bytes or 1024 bytes?

See https://en.wikipedia.org/wiki/Byte#Multiple-byte_units for a discussion.

I'm not sure what convention PyTorch or NVIDIA uses (if any). To be as clear as possible I would use "KiB" (and "MiB", ...) to refer to kibibytes (mebibytes, ...) explicitly.